fleetspeak_python/setup.py

```diff
@@ -85,7 +85,7 @@ def compile_protos():
     # grpcio-tools and then uninstalled when grr-response-proto's setup.py runs
     # and reinstalled to the version required by grr-response-proto.
     subprocess.check_call(
-        [sys.executable, "-m", "pip", "install", GRPCIO_TOOLS])
+        [sys.executable, "-m", "pip", "install", "grpcio-tools" + GRPCIO_VERSION])
```

Comment:
I suggest f"grpcio-tools{GRPCIO_VERSION}" here and below.

Reply:
I'll update it, thanks for the suggestion.

I always found fstrings to be rather impractical to read if concatenation is simple, is there a particular reason for your suggestion?